### PR TITLE
Add security page

### LINKED ISSUE #2389: /security - overview build
Brief: https://docs.google.com/document/d/1Fbfsrmo0-tFSgXWSz8Vi-wBOftpKTrzYcGr6bu7w6rA/edit
Copy doc: https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add new security page

Todo: Find out where this is going to live in the UI

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/security](http://0.0.0.0:8001/security)
- Make sure the page is as per [copy doc](https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit#)

## Issue / Card

Fixes #2389 Fixes https://github.com/canonical-websites/www.ubuntu.com/pull/2495
Doc: https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit#

### CODE REVIEW

**Comment by frankiegranato** [comment]
We need to add the 'Find out more' section.
Lets try to add the heart pictogram next to 'Canonical puts security at the heart of Ubuntu'
Lets try to use the medal graphic for the 'CESG' row
Can we try something different for the 'Helping you manage security' section - Lets try it on cards and if that doesn't work, maybe try it on an aubergine background? Or in 'normal' rows?

**Comment by grahambancroft** [comment]
@frankiegranato All done

File: templates/security/index.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+  </div>
+</section>
+
+<section class="p-strip--light is-deep u-image-position is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h2>Find out why the UK Government puts Ubuntu in first place for security</h2>
+      <p><abbr title="Communications-Electronics Security Group">CESG</abbr>, the security arm of the UK government rated Ubuntu as the most secure operating system of the 11 they tested.</p>
+      <p>For the first time, both a <abbr title="Defence Industry Security Association">DISA</abbr> approved <abbr title="Security Technical Implementation Guide">STIG</abbr> and a <abbr title="Center for Internet Security">CIS</abbr> Benchmark are available for Ubuntu 16.04 LTS. This is in addition to the CIS Benchmark already available for 14.04 LTS.</p>
+      <a class="p-button--positive" href="https://insights.ubuntu.com/2014/01/10/ubuntu-scores-highest-in-uk-gov-security-assessment/?_ga=2.112079751.2078786262.1509620345-1171504626.1506072082" class="p-link--external">Read the <span class="u-off-screen">UK Gov Report Summary </span>case study</a>
```
Comment (anthonydillon) [inline templates/security/index.html]: Two classes

File: templates/security/index.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+      <div class='p-heading-icon'>
+        <div class='p-heading-icon__header u-no-margin--bottom'>
+          <img class='p-heading-icon__img' src='{{ ASSET_SERVER_URL }}53616a73-pictogram-heart-orange.svg' alt='' />
+          <h2 class="p-heading-icon__title">Canonical puts security at the heart of Ubuntu</h>
```
[the remark this answers is not in the document]
Comment (grahambancroft) [reply templates/security/index.html]: Done

File: templates/security/index.html [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+        </div>
+      </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <ul class="p-matrix is-trisected">
+        <li class="p-matrix__item">
+          <div>
+            <h3 class="p-matrix__title">Fast fixes</h3>
```
[the remark this answers is not in the document]
Comment (grahambancroft) [reply templates/security/index.html]: Done

File: templates/security/index.html [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+            <h3 class="p-matrix__title">Secure snap packages</h3>
+            <p class="p-matrix__desc">Software packages delivered as strict-mode snaps are fully confined using AppArmor, device cgroups, and seccomp.</p>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+{% include "shared/_case-study-itstrategen.html" %}
```
[the remark this answers is not in the document]
Comment (grahambancroft) [reply templates/security/index.html]: Done

File: templates/security/index.html [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
+        <h2 class="p-muted-heading u-align--center">Ubuntu is trusted by</h2>
+        <ul class="p-inline-images">
```
[the remark this answers is not in the document]
Comment (grahambancroft) [reply templates/security/index.html]: What's DT?

File: templates/security/index.html (hunk 1 above)
Comment (grahambancroft) [reply templates/security/index.html]: Done

File: templates/security/index.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (grahambancroft) [reply templates/security/index.html]: Done

**Comment by frankiegranato** [comment]
- [ ] Can we try the heart on the right of the heading please? 

- [ ] CESG section: Can we have just one medal (the Ubuntu one)

- [ ] I think there is too much padding in the 'resources' row. (this is an old problem I think)

- [ ] Add heading above 'resources' row: Learn more about Ubuntu security (see my edit in the copydoc)

- [ ] Re. the secondary cta in the last row, can we try a see through button next to the white button. 

Apologies for all the noodling. I think we might need some design help with this page :-( Lets see how it looks after this round of changes and see if we need to rope Spencer or Karlos in.

**Comment by grahambancroft** [comment]
@frankiegranato Couldn't find a see through button, will try some more on Wednesday

**Comment by anthonydillon** [comment]
@grahambancroft is base button: https://docs.vanillaframework.io/en/patterns/buttons#base-button

**Comment by grahambancroft** [comment]
@anthonydillon I tried that one but it was weird on hover on an accent background, wanted to debug a little more before filing an issue

**Comment by spencerbygraves** [comment]
> Can we try the heart on the right of the heading please?

Having a look around, when we have an icon of that size next to a h2 it goes before the text.

> Re. the secondary cta in the last row, can we try a see through button next to the white button.

Perhaps just move the link up to the right of the button? The base button on a background colour may need some work.

**Comment by spencerbygraves** [comment]
Can we swap the security padlock and chain image with the pictogram please:

https://assets.ubuntu.com/v1/16e21217-pictogram-safety-orange.svg

Thanks

**Comment by spencerbygraves** [comment]
@anthonydillon thanks. Green btn at the top has gone funny, and external link at the bottom is blue.

**Comment by anthonydillon** [comment]
@spencerbygraves Could you try hard refresh, they both seem fine to me.

**Comment by frankiegranato** [comment]
The 'UBUNTU IS TRUSTED BY' heading should be centred.

**Comment by frankiegranato** [comment]
screen shot 2017-12-08 at 11 55 03
In the future, we do need to fix this issue that happens when external links go to two lines and the external link icon touches the baseline of the line above. I'll file a bug.

**Comment by anthonydillon** [comment]
@spencerbygraves @frankiegranato all done.

**Pull request merged**